sdk/eventhub/event-hubs/rollup.base.config.js

```diff
@@ -75,9 +77,9 @@ export function browserConfig(test = false) {
     input: input,
     external: ["ms-rest-js"],
     output: {
-      file: "browser/index.js",
+      file: "browser/event-hub.js",
       format: "umd",
-      name: "ExampleClient",
+      name: "Azure.Messaging.EventHub",
```

Comment:
I believe this should be `browser/event-hubs.js` and `Azure.Messaging.EventHubs`. When referring to the service or the library, we use "hubs". When referring to the client which deals with a single entity, we use `hub`

Reply:
Oh, looks like most of the changes didn't actually get added - will take a look and push again.

Reply:
Nvm, it's as intended - will update to use 'hubs'